marconi-chain-index/src/Marconi/ChainIndex/Indexers.hs

```diff
@@ -127,9 +127,11 @@ buildIndexers securityParam catchupConfig utxoConfig mintEventConfig epochStateC
       [utxoWorker, spentWorker, datumWorker, mintTokenWorker]
 
   utxoQueryIndexer <-
-    lift $
-      UtxoQuery.mkUtxoSQLiteQuery $
-        UtxoQuery.UtxoQueryAggregate utxoMVar spentMVar datumMVar blockInfoMVar
+    Core.withTrace mainLogger
```

Comment:
Same as in the other PR, append a name to `mainLogger` here to differentiate the events source.